### PR TITLE
Safe watch real checks

### PR DESCRIPTION
**Pull request by postwait**

### CODE REVIEW

File: src/noit_check.c [hunk 1]
```diff
@@ -981,7 +981,14 @@ noit_check_watch(uuid_t in, int period) {
 
   mtevL(noit_debug, "noit_check_watch(%s,%d)\n", uuid_str, period);
   if(period == 0) {
-    return noit_poller_lookup(in);
+    uuid_copy(n.checkid, in);
```
Comment (pamaddox) [inline src/noit_check.c]: Use `mtev_uuid_copy`, it's much faster.

Comment (postwait) [inline src/noit_check.c]: There are 17 uuid_copy's in the code, perhaps we can change them in an unrelated commit. This is just moving existing code around.

Comment (pamaddox) [inline src/noit_check.c]: That's reasonable. Changed to an approval.
